### DISCUSSION TITLE
I want to be able to see reviewers on PRs from the menu page

### DISCUSSION THREAD

**Discussion by JackWebbTelus** [community/community#52631] [Pull Requests]
### Select Topic Area

Product Feedback

### Body

When you are checking PRs for review, you can't see who has already been assigned to review a PR without opening each PR individually, I would like to be able to see PR reviewers the way you can see assignees on the menu 

Azure DevOps has this feature and it is a simple but effective improvement

![image](https://user-images.githubusercontent.com/123937950/231478012-509fc567-63e9-45c7-8917-7afdbf856697.png)

**Comment by shawnmcknight** [comment]
We just migrated from Bitbucket to Github.  Although Github is significantly better in a lot of ways, the lack of being able to easily see the reviewers is a big miss in my opinion and definitely a regression from what we're used to.

Doing a quick scan in this discussions topic I found 2 other suggestions that were basically the same thing as this one:

* #17683 
* #37961

This seems like a great usability improvement and would be very nice to see!

**Comment by JackWebbTelus** [comment]
Hi Shawn,

I also made this request

https://github.com/orgs/community/discussions/52631

Kind regards,
Jack

On Fri, Apr 14, 2023 at 9:06 PM Shawn McKnight ***@***.***>
wrote:

> We just migrated from Bitbucket to Github. Although Github is
> significantly better in a lot of ways, the lack of being able to easily see
> the reviewers is a big miss in my opinion and definitely a regression from
> what we're used to.
>
> Doing a quick scan in this discussions topic I found 2 other suggestions
> that were basically the same thing as this one:
>
>    - #17683 <https://github.com/orgs/community/discussions/17683>
>    - #37961 <https://github.com/orgs/community/discussions/37961>
>
> This seems like a great usability improvement and would be very nice to
> see!
>
> —
> Reply to this email directly, view it on GitHub
> <https://github.com/orgs/community/discussions/52631#discussioncomment-5619895>,
> or unsubscribe
> <https://github.com/notifications/unsubscribe-auth/A5RSJHTZ2BTFJFG7PEUVAADXBGU2ZANCNFSM6AAAAAAW3W5BAY>
> .
> You are receiving this because you authored the thread.Message ID:
> ***@***.***>
>

-- 

CONFIDENTIALITY NOTICE: This email, including any attachments, is for the 
sole use of the intended recipient and may contain confidential 
information. If you are not the intended recipient, please notify us 
immediately and destroy this email and any copies.

**Comment by yuvalhermelin-fijoya** [comment]
Why are there no replies on this from the github product team 🤷 

Guys - please take a look at this :) It is an addition that is not too hard and potentially very beneficial to some teams. Notice there are 21 people above that agree with the original author - and I personally have heard a bunch of people asking for this verbally (I am sure many people want this but just have not asked)

Much appreciated!

**Reply by yuvalhermelin-fijoya** [reply]
@ebndev @queenofcorgis I would appreciate it if y'all could help get this seen and prioritized at some point :)

**Comment by LarsPeterson** [comment]
Yes BitBucket has had this feature forever. Makes all the difference in the world.

![image](https://github.com/user-attachments/assets/4750ab20-a630-4e2e-a3b2-f6306a405fe5)
Notice the green checkmarks next to the reviewers ^

They have it on the "menu" as well.

It's that simple. Please add this feature, big deal for teams who do PR reviews.

EDIT

After doing more research, you can install these Chrome extensions which certainly help.

https://chromewebstore.google.com/detail/enhanced-github/anlikcnbgdeidpacdbdljnabclhahhmd?hl=en
https://chromewebstore.google.com/detail/refined-github/hlepfoohegkhhmjieoechaddaejaokhf

![image](https://github.com/user-attachments/assets/7a163963-375a-463d-ac28-266f3f99317f)

**Reply by yuvalhermelin-fijoya** [reply]
@ebndev @queenofcorgis  Did y'all see this thread? Waiting for a reply 🙏

**Reply by ebndev** [reply]
Hi @yuvalhermelin-fijoya, we've brought this to the attention of the Community Manager of this category so they can assess the feedback and forward it to the proper GitHub team.

**Here's what you can expect moving forward ⏩**
- Your input will be carefully reviewed and cataloged by members of our product teams. 
  - Due to the high volume of submissions, we may not always be able to provide individual responses.
  - Rest assured, your feedback will help chart our course for product improvements.
- Other users may engage with your post, sharing their own perspectives or experiences. 
- GitHub staff may reach out for further clarification or insight. 
  - We may 'Answer' your discussion if there is a current solution, workaround, or roadmap/changelog post related to the feedback.

**Where to look to see what's shipping 👀**
- Read the [Changelog](https://github.blog/changelog/) for real-time updates on the latest GitHub features, enhancements, and calls for feedback.
- Explore our [Product Roadmap](https://github.com/orgs/github/projects/4247), which details upcoming major releases and initiatives.

**What you can do in the meantime 💻**
- Upvote and comment on other user feedback Discussions that resonate with you.
- Add more information at any point! Useful details include: use cases, relevant labels, desired outcomes, and any accompanying screenshots.

As a member of the GitHub community, your participation is essential. While we can't promise that every suggestion will be implemented, we want to emphasize that your feedback is instrumental in guiding our decisions and priorities. ⭐

**Reply by lukewarkotsch** [reply]
@ebndev @queenofcorgis  any movement on this request? This feature seems like a no brainer and is a massive gap in functionality between bitbucket and github. I don't see how anyone can really make full use of the pull request lists without this feature. How are people reviewing their lists at a glance? Are we expected to open every single PR individually in order to see who has reviewed it?

**Comment by sarah-gelt** [comment]
I miss this so much from using Bitbucket at a previous company and it would speed up our day-to-day work a massive amount. Upvoted 👍  💯

**Comment by BadisMaalej** [comment]
Upvoted ! This is a must have 👍

**Comment by yuvalhermelin-fijoya** [comment]
@ebndev @queenofcorgis Can you check in with the product team? Its been almost two years (and half a year since you guys last replied to this thread) - and I feel like this is a quick win. Doesn't seem too difficult to implement and would help a lot of people. Does product not want it?

**Comment by elliottkember** [comment]
I would really appreciate this feature. It's difficult to know which PRs don't have reviewers assigned to them without opening them all one by one.

**Comment by Austinae** [comment]
gta 6 will come out before this feature is released

**Comment by jeromeelliott** [comment]
Having just migrated from Bitbucket to Github, this is a huge feature gap and negatively impacting our productivity, having to open every open PR each morning to get a sense of where it is in our team's signoff process.

The [screenshot in this post](https://github.com/orgs/community/discussions/52631#discussioncomment-10453964) shows how smooth and valuable this view is in a list of PRs.

Please please please consider adding this soon.

**Comment by bbrks** [comment]
Any updates on this? It's far too easy to miss PRs that have been sat on me awaiting review but not explicitly assigned.

**Comment by hannahlizw** [comment]
+1 like this is such a simple feature.. and it was so nice in bitbucket

**Comment by jamesvrt** [comment]
Bump. This feature would be so helpful for triaging open PRs.

**Comment by martin-knopf** [comment]
I need this.

**Comment by thilo-dual** [comment]
I need this
